celery/backends/redis.py

```diff
@@ -198,6 +203,27 @@ def __init__(self, host=None, port=None, db=None, password=None,
 
         if url:
             self.connparams = self._params_from_url(url, self.connparams)
+
+        # If we've received SSL parameters via query string or the
+        # redis_backend_use_ssl dict, check ssl_cert_reqs is valid. If set
+        # via query string ssl_cert_reqs will be a string so convert it here
+        if ('connection_class' in self.connparams and
+                self.connparams['connection_class'] == redis.SSLConnection):
```

Comment:
Equality check will work too but since this is a class, let's use the `is` operator.

Reply:
Update committed. `flake8` re-run along with unit tests for all supported Python versions. Many thanks for reviewing.